### PR TITLE
Meal planner generated in html

### PR DESCRIPTION
**Pull request by monwd**

### CODE REVIEW

File: meal_planner_classes.py [hunk 1]
```diff
@@ -26,7 +27,7 @@ def __init__(self, df_recipes):
     def draw_meal_type(self, meal_type):
         meal = self.df_recipes.loc[df_recipes['Type'] == meal_type].sample()
         df_recipes.drop(meal.index[0], inplace=True)
-        meal1 = Meal(meal.index[0], meal.Type[0], meal.Category[0], meal.Ingredients[0])
+        meal1 = Meal(meal.index[0], meal.iloc[0, 0], meal.iloc[0, 1], meal.iloc[0, 2])
```
Comment (milkisbad) [inline meal_planner_classes.py:30]: this reverts change in previous mr?

File: meal_planner_classes.py [hunk 2]
```diff
@@ -59,6 +86,7 @@ class WeeklyPlan:
     def __init__(self, num_of_days):
         self.num_of_days = num_of_days
         self.list_of_day_plans = None
+        self.table_html = None
```
Comment (milkisbad) [inline meal_planner_classes.py:89]: no need to keep `table_html` as attribute. Also 1 line to remove in `table` function

File: meal_planner_classes.py [hunk 3]
```diff
@@ -70,8 +98,51 @@ def draw_meal_plan(self):
...
+        self.table_html = table_html
+
+        return table_html
+
+    def table_display(self):
+        display(HTML(self.table_html))
+
     def __str__(self):
-        return '\n'.join([str(plan) for plan in self.list_of_day_plans])
+        return '\n'.join([str(plans) for plans in self.list_of_day_plans])
```
Comment (milkisbad) [inline meal_planner_classes.py:145]: well shouldn't this stay as `plan`?

File: meal_planner_classes.py [hunk 4]
```diff
@@ -89,4 +160,5 @@ def get_input():
     num_of_days = get_input()
     w = WeeklyPlan(num_of_days)
     w.draw_meal_plan()
-    print(w)
+    w.table()
+    print(w.table_display())
```
Comment (milkisbad) [inline meal_planner_classes.py:164]: if anything just do `print(w.table())` in here

File: meal_planner_classes.py [hunk 5]
```diff
@@ -70,8 +98,51 @@ def draw_meal_plan(self):
         self.list_of_day_plans = list_of_day_plans
         return list_of_day_plans
 
+    def table(self):
```
Comment (milkisbad) [inline meal_planner_classes.py:101]: rename this to sth like `create_html_table`

File: meal_planner_classes.py [hunk 6]
```diff
@@ -70,8 +98,51 @@ def draw_meal_plan(self):
...
+        w1 = WeeklyPlan(num_of_days)
+        plans = self.list_of_day_plans
+        days_str = f"{'</th><th>'.join(list(map(lambda plan: plan.html_date(), plans)))}"
+        plans_b = f"{'</td><td>'.join(list(map(lambda plan: plan.html_meal(meal=plan.breakfast), plans)))}"
```
Comment (milkisbad) [inline meal_planner_classes.py:105]: rename to sth like `html_row_breakfasts`

File: meal_planner_classes.py [hunk 7]
```diff
@@ -70,8 +98,51 @@ def draw_meal_plan(self):
...
+        <tr>
+       <td style="width: 10; height: 18px; text-align: left;">
+<strong>DINNER</strong></span>
+</td>
+        <td > {plans_d}</td>
+        </tr>
+        </tbody>
+        </table>
+        
+         """
```
Comment (milkisbad) [inline meal_planner_classes.py:136]: formatting is a bit screwed on this one is it only on github?

File: meal_planner_classes.py (hunk 6 above)
Comment (milkisbad) [reply meal_planner_classes.py:105]: change the name for all variables here

File: meal_planner_classes.py (hunk 1 above)
Comment (monwd) [reply meal_planner_classes.py:30]: not really sure from where this change came. Corrected.

File: meal_planner_classes.py (hunk 2 above)
Comment (monwd) [reply meal_planner_classes.py:89]: If I remove `self.table_html` from `table` function , I will get AttributeError 'WeeklyPlan' object has no attribute 'table_html' raised when system tries to run `display(HTML(self.table_html`)

File: meal_planner_classes.py (hunk 4 above)
Comment (monwd) [reply meal_planner_classes.py:164]: I assumed that we want to have just html view of our table , but added `print(w.table())` as well

File: meal_planner_classes.py (hunk 2 above)
Comment (milkisbad) [reply meal_planner_classes.py:89]: yes, the idea is to make a flow like this
`html_str = w.table()`
`w.display_html(html_str)`
then displaying html would happen only when you need it.

File: meal_planner_classes.py [hunk 8]
```diff
@@ -70,6 +97,40 @@ def draw_meal_plan(self):
...
+        <td style="width: 10; height: 18px; text-align: left;"><strong>DINNER</strong></span></td>
+        <td>{html_row_dinner}</td>
+        </tr>
+        </tbody>
+        </table>
+        
+         """
+        return table_html
+
+    def table_display(self, html_str):
```
Comment (milkisbad) [inline meal_planner_classes.py:131]: rename to sth like `display_html` after its done, you can merge to master

**Pull request merged**